Tests/XMLCoderTests/ErrorContextTests.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  ErrorContextTests.swift
+//  XMLCoder
+//
+//  Created by Matvii Hodovaniuk on 1/7/19.
+//
+
+import Foundation
```

Comment:
was this mostly file committed by mistake?

Reply:
@MaxDesiatov yes, I can't see this file through Xcode. It was committed by mistake.